karma.conf.js

```diff
@@ -25,10 +25,6 @@ module.exports = function(config) {
 
       // Moment.js
       "www/lib/moment/min/moment.min.js",
-
-      // Ionic Tinder card library -->
-      "www/lib/collide/collide.js",
-      "www/lib/ionic-ion-tinder-cards/ionic.tdcards.js",
       
```

Comment:
Hi @svm96 , I understand that these 2 lines cause problem in your testing process.
But this problem can be solved by running **bower install** in your computer (refer to README for more details).

In fact, you never remove lines here. Adding more lines is OK (in case you need more Bower library)

Can you write back these 2 lines into karma.conf.js and the run **bower install**?